package.json

```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-hooks-use-modal": "^2.1.0"
```

Comment:
this shouldn't be here in root package.json : restore package.json and package-lock.json